src/Microsoft.TestPlatform.AdapterUtilities/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -1,27 +1,28 @@
 Microsoft.TestPlatform.AdapterUtilities.HierarchyConstants
 Microsoft.TestPlatform.AdapterUtilities.HierarchyConstants.Levels
 Microsoft.TestPlatform.AdapterUtilities.ManagedNameConstants
 Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.InvalidManagedNameException
-Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.InvalidManagedNameException.InvalidManagedNameException(string message) -> void
+Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.InvalidManagedNameException.InvalidManagedNameException(string? message) -> void
 Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper
 Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameParser
 Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameParser.ManagedNameParser() -> void
 Microsoft.TestPlatform.AdapterUtilities.TestIdProvider
-Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.AppendString(string str) -> void
-Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.GetHash() -> byte[]
+Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.AppendString(string! str) -> void
+Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.GetHash() -> byte[]!
 Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.GetId() -> System.Guid
 Microsoft.TestPlatform.AdapterUtilities.TestIdProvider.TestIdProvider() -> void
-static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper.GetManagedName(System.Reflection.MethodBase method, out string managedTypeName, out string managedMethodName) -> void
-static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper.GetManagedName(System.Reflection.MethodBase method, out string managedTypeName, out string managedMethodName, out string[] hierarchyValues) -> void
-static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper.GetMethod(System.Reflection.Assembly assembly, string managedTypeName, string managedMethodName) -> System.Reflection.MethodBase
-static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameParser.ParseManagedMethodName(string managedMethodName, out string methodName, out int arity, out string[] parameterTypes) -> void
-static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameParser.ParseManagedTypeName(string managedTypeName, out string namespaceName, out string typeName) -> void
\ No newline at end of file
+static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper.GetManagedName(System.Reflection.MethodBase! method, out string! managedTypeName, out string! managedMethodName) -> void
+static Microsoft.TestPlatform.AdapterUtilities.ManagedNameUtilities.ManagedNameHelper.GetManagedName(System.Reflection.MethodBase! method, out string! managedTypeName, out string! managedMethodName, out string![]? hierarchyValues) -> void
```

Comment:
this will always be an array of string with a length of `Microsoft.TestPlatform.AdapterUtilities.HierarchyConstants.Levels.TotalLevelCount`. It can't be null; one or more items can be.